tests/test_telemetry.py

```diff
@@ -359,3 +359,42 @@ def test_app_dependencies_loaded(self):
         for dependency, seen in seen_loaded_dependencies.items():
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
+
+    def test_app_started_product_info(self):
+        """Assert that product information is accurately reported by telemetry"""
+
+        def validator(data):
+            if data["request"]["content"].get("request_type") == "app-started":
+                content = data["request"]["content"]
+                if content.get("request_type") == "app-product-change":
+                    products = content["payload"]["products"]
+                    try:
+                        products["appsec"]
+                    except KeyError:
+                        raise Exception(
+                            "Product information is not accurately reported by telemetry on app-started event"
+                        )
+
+            self.validate_library_telemetry_data(validator)
+
+    def test_app_started_client_configuration(self):
+        """Assert that default and other configurations that are applied upon start time are sent with the app-started event"""
+        configurationMap = map(
+            "DD_AGENT_HOST", "DD_TRACE_AGENT_PORT", "DD_APPSEC_ENABLED", "DD_TELEMETRY_HEARTBEAT_INTERVAL"
+        )
+
+        def validator(data):
+            if data["request"]["content"].get("request_type") == "app-started":
+                content = data["request"]["content"]
+                if content.get("request_type") == "app-client-configuration":
+                    cnt = 0
+                    configurations = content["payload"]["conf_key_values"]
```

Comment:
you want to test the presence of `"DD_AGENT_HOST", "DD_TRACE_AGENT_PORT", "DD_APPSEC_ENABLED", "DD_TELEMETRY_HEARTBEAT_INTERVAL"` in `conf_key_values`, right ? is yes, same comment as above.